indico/MaKaC/services/implementation/registration.py

```diff
@@ -101,8 +102,9 @@ def _getAnswer(self):
                    "date": format_date(self._conf.getAdjustedStartDate()),
                    "server": {"baseUrl": baseURL,
                               "consumerKey": checkin_app.client_id,
-                              "consumerSecret": checkin_app.client_secret,
-                              }
+                              "auth_url": url_for('oauth.oauth_authorize'),
+                              "token_url": url_for('oauth.oauth_token')
+                              },
```

Comment:
broken indentation (when aligning keys, the closing `}` and if used the `,` right after it go in the previous line)